src/components/m-table-toolbar.js

```diff
@@ -53,6 +57,26 @@ export class MTableToolbar extends React.Component {
       .exportFile();
   }
 
+  defaultExportPdf = () => {
+    const [columns, data] = this.getTableData();
+
+    let content = {
+      startY: 50,
+      head: [columns.map(columnDef => columnDef.title)],
+      body: data
+    };
+
+    const unit = "pt";
+    const size = "A4"; 
+    const orientation = "landscape";
+
+    const doc = new jsPDF(orientation, unit, size);
+    doc.setFontSize(15);
+    doc.text(this.props.title, 40, 40);
+    doc.autoTable(content);
+    doc.save((this.props.exportFileName || this.props.title || 'data') + '.pdf');
+  }
+
   exportCsv = () => {
     if (this.props.exportCsv) {
       this.props.exportCsv(this.props.columns, this.props.data);
```

Comment:
Why defaultExportCsv call removed?

Reply:
No worries.  Its in m-table-toolbar.js on line 49. Let me know if you need anything else!